nilearn/plotting/find_cuts.py

```diff
@@ -76,6 +76,8 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
         activation_threshold = fast_abs_percentile(my_map[my_map != 0].ravel(),
                                                    80)
     mask = np.abs(my_map) > activation_threshold - 1.e-15
+    if mask.sum() == 0:
```

Comment:
For some obscure reason the 'mask' happened to contain only zeros for a certain nifti image, which is why largest_connected_component crashed. Unfortunately, I have not succeeded in creating the same situation in a test :(